### ISSUE TITLE
deploy with codeanywhere.com

### ISSUE THREAD

**Issue by Bones80** [jpillora/node-torrent-cloud#23]
How can I get this to deploy with codeanywhere.com?

thanks

**Comment by rakshans1** [comment]
Try to deploy in heroku..it is free

**Comment by Bones80** [comment]
I don't think heroku works anymore with this and codeanywhere is free too.

**Comment by nk932714** [comment]
yep heroku is not working anymore with this repo. Heroku is suspending this app immediately after deploying.

About codeanywhere in that site repo is deploying but downloading is not starting i don't know what is the issue with downloading on codeanywhere

Also on openshift downloading is not starting

**Comment by rakshans1** [comment]
@nk932714 maybe they disabled torrent download in network
